azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/header/InfoHeaderViewModel.kt

```diff
@@ -54,6 +54,21 @@ internal class InfoHeaderViewModel(
         }
     }
 
+    /* <CUSTOM_CALL_HEADER> */
+    fun getFormattedElapsedDuration(formatString: String): String {
+        val elapsedDuration = callDurationManager?.getElapsedDuration() ?: 0L
+        val seconds = (elapsedDuration / 1000) % 60
+        val minutes = (elapsedDuration / (1000 * 60)) % 60
+        val hours = (elapsedDuration / (1000 * 60 * 60)) % 24
+
+        return when {
+            hours > 0 -> String.format(formatString, hours, minutes, seconds)
```

Comment:
Will this work in hours/minutes, has it been tested? I think we need 3 string resources.
One for H:M:S, One for M:S, and one for S.

<string name="azure_communication_ui_calling_view_info_header_call_timer_format_with_hours" translatable="false">%02d:%02d:%02d</string>
<string name="azure_communication_ui_calling_view_info_header_call_timer_format_with_minutes" translatable="false">%02d:%02d</string>
<string name="azure_communication_ui_calling_view_info_header_call_timer_format_with_seconds" translatable="false">%02d</string>

Reply:
Also, is possible to add unit test here on ViewModel method?